docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
+    ```
+6. To test your configuration setup, you can run `aws-nuke --config nuke.yaml` to list all nukeable resources. You need to add the flag `--no-dry-run` to actually delete resources.
+7. When you want to clean your account, run `aws-nuke --config nuke.yaml --no-dry-run`. You will be asked for confirmation twice and need to enter the account alias you chose when you created the account (eg. `jdoe-dev`).
+
+## Try running `bin/setup`
+1. From within your local fork of `civiform-deploy`, run `bin/setup` to kick off the setup script. At a very high level this script does the following things:
+    - Creates a `checkout` folder in your local copy of `civiform-deploy` and pulls in the code from `cloud-deploy-infra` that contains all of the actual implementation for deployment.
+    - Runs the scripts pulled in from `cloud-deploy-infra` to deploy the instance.
+    - Creates an S3 bucket in Amazon in which to store resources.
+    - Creates all of the resources specified in the Terraform config files in `cloud-deploy-infra`. (Note: this step is the same as running the `bin/deploy` script.) Terraform will prompt you to type "yes" several times to confirm creation of resources.
+    - Automatically retries resource creation once because of a [known bug with Terraform and AWS](https://github.com/hashicorp/terraform-provider-aws/issues/19583)
+    - Prompts you to create several secrets like for login integration credentials. These strings don't matter unless you are testing applicant authentication. They can also be changed later via the AWS console.
+    - Starts the server.
+    - Checks for service health.
+    - If everything is successful, the script will print out `Server is available on url:....` and give you a url to add to your domain registrar.
+2. Create a new record in the [domain registrar](https://domains.google.com/registrar/civiform.dev/dns) with the following values and click "Save":
+    - Host name: The domain name you set when requesting your certificate (step 5 in Configure new AWS Account above)
+    - Type: CNAME
+    - TTL: 3600
+    - Data: URL copied from script output (will look something like `jdoe-dev-civiform-lb-<some_numbers>.us-east-1.elb.amazonaws.com`)
+3. Wait a few minutes and then load up your domain url and you should have a live site!!
+
+Note: You can use the Server url directly and skip the step of adding it to the domain registrar if you like. You will receive an `SSL_ERROR_BAD_CERT_DOMAIN` error because the certificate is only valid for the domain you defined when you created it, but if you click "Advanced" on the error page it should give you an option to proceed anyway.
+
+Troubleshooting:
+- If you get an error message about "Amaxon 3 already exists", this is likely because you have run the script before (possibly unsuccessfully) and some resources were created. Run `aws-nuke --config nuke.yaml --no-dry-run` to clear out all resources and try again.
```

Comment:
```suggestion
- If you get an error message about "Amazon 3 already exists", this is likely because you have run the script before (possibly unsuccessfully) and some resources were created. Run `aws-nuke --config nuke.yaml --no-dry-run` to clear out all resources and try again.
```